src/app/(proper_react)/redesign/(authenticated)/admin/feature-flags/AddFeatureFlag.tsx

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+export const AddFeatureFlag = () => {
+  // @ts-ignore FIXME determine correct event type
+  const handleSubmit = void (async (event) => {
```

Comment:
Hm, `next lint` accepted this but it's actually wrong and the handler isn't being called so the UI isn't working... I'll figure this out.